File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.java
class Solution {
    public int maxProfit(int[] prices) {
        int size = prices.length;
        int rightMax = prices[size - 1];

        int profit = 0;
        int pointer = size - 2;

        while (pointer > -1) {
            int curr = prices[pointer];
            if(curr > rightMax) {
                rightMax = curr;
            } else {
                profit = Math.max(profit, rightMax - curr);
            }
            pointer--;
        }
        return profit;
    }
}

File: Data Structures & Algorithms/ugly-number/submission-0.py
class Solution:
    def isUgly(self, n: int) -> bool:
        if n == 1: return True
        while n:
            if n % 2 == 0:
                n = n / 2
                continue
            if n % 3 == 0:
                n = n / 3
                continue
            if n % 5 == 0:
                n = n / 5
                continue
            if n == 1:
                break
            else:
                return False
        return True